python/python/lance/ray/sink.py

```diff
@@ -74,13 +75,18 @@ def record_batch_converter():
             tbl = _pd_to_arrow(block, schema)
             yield from tbl.to_batches()
 
+    max_bytes_per_file = (
+        DEFAULT_MAX_BYTES_PER_FILE if max_bytes_per_file is None else max_bytes_per_file
+    )
```

Comment:
Why not just put `DEFAULT_MAX_BYTES_PER_FILE` in the signature as the default value?

Reply:
So using None here we can later detect whether users specify this value or not. During a benchmark, the default 90GB causes OOM (now we know it was [a bug in arrow](https://github.com/apache/arrow-rs/pull/5347). This allows us to provide a better value later.

Reply:
I see. Seems like a good thing to write a `TODO` comment for.